iphone/Classes/TiUIListView.m

```diff
@@ -201,6 +201,11 @@ - (UITableView *)tableView
     if ([_tableView superview] != self) {
         [self addSubview:_tableView];
     }
+    
+    if ([TiUtils isIOS9OrGreater]) {
+        _tableView.cellLayoutMarginsFollowReadableWidth = NO;
+    }
+    
     return _tableView;
```

Comment:
Please do this check before line 200 in the initial creation of the ``UITableView``. Also make sure to use the ``IS_XCODE_7`` macro to not break Xcode 6 builds.